### PR TITLE
[WIP] Update docs to nhost-js-sdk v3

### PR DESCRIPTION
**Pull request by nunopato**
- update docs for the nhost-js-sdk v3 release

### CODE REVIEW

File: src/pages/libraries/nhost-js-sdk.mdx [hunk 1]
```diff
@@ -21,210 +21,203 @@ $ yarn add nhost-js-sdk
...
-  use_cookies: true,
-};
-```
+| Parameters     |               |          |                                                                                         |   |
+|--------------------------|---------------|----------|-----------------------------------------------------------------------------------------|---|
+| *baseURL*                  | string        | required | Backend URL                                                                             |   |
+| *useCookies*               | boolean       | optional | Use cookies for auth (defaults to `false`)                                              |   |
+| *ssr*                      | boolean       | optional | Enable server-side rendering mode (defaults to `typeof window === "undefined"`) |   |
+| *autoLogin*                | boolean       | optional | Auto logins user (defaults to `true`)                                                     |   |
+| *clientStorage*            | ClientStorage | optional | Storage to use (defaults to `Window.localStorage`)                                      |   |
```
Comment (elitan) [inline src/pages/libraries/nhost-js-sdk.mdx:43]: lower case on `Window`.

File: src/pages/libraries/nhost-js-sdk.mdx [hunk 2]
```diff
@@ -21,210 +21,203 @@ $ yarn add nhost-js-sdk
...
 ```js
-auth.changeEmailChange(ticket);
+auth.confirmEmailChange(ticket);
 ```
 
-### Change password
+For changing a user's password use `changePassword()`
 
 ```js
 auth.changePassword(old_password, new_password);
```
Comment (elitan) [inline src/pages/libraries/nhost-js-sdk.mdx:140]: `old_password` => `oldPassword`.

**Review by elitan — CHANGES_REQUESTED** [review]
Also, the current `code`-blocks look like links in dark mode.

File: src/pages/libraries/nhost-js-sdk.mdx (hunk 1 above)
Comment (nunopato) [reply src/pages/libraries/nhost-js-sdk.mdx:43]: 👍

File: src/pages/libraries/nhost-js-sdk.mdx (hunk 2 above)
Comment (nunopato) [reply src/pages/libraries/nhost-js-sdk.mdx:140]: 👍

**Review by elitan — APPROVED** [review]

**Pull request merged**